script/QA/QA_22.txt

```diff
@@ -51,277 +57,313 @@ so please skip this section and go back to the main story.
 }
 [sha:3aa0c83200e81bfdf86f9bd64b0b4d5a73901ae1]{
 -- Page 232, Offset 43746.
 -- 『地球にとっては重くないので、
-'It is nothing compared to the weight of Earth,
+It isn't heavy as far as the Earth is concerned,
 }
 [sha:767b749f87e462ef7f1b42d68f5453d9072fcce9]{
 -- Page 232, Offset 43747.
 -- 地形には重さによるダメージはかからない』
-so the weight of the damage will not reach.'
+ and so%{n}
+the terrain will not take any crush damage.
 }
 [sha:f170fd16f53142c1935e3453be6c26d9a46b770f]{
 -- Page 234, Offset 43748.
 -- となります。地面は陥没しません。
-or something similar. The ground will not collapse.
+Or something along those lines, anyway.%{n}
 }
 [sha:3f3788ed1c1aea1ea23a69cf4fd7f5183067a122]{
 -- Page 234, Offset 43749.
 -- ただし人工物はその限りではありません。
-However, this does not apply to man-made objects.
+However, though the ground will not cave in,%{n}
+the same can't be said for man-made objects.
 }
 [sha:82dd33cb70a5547dc53ca1403a31dd290602a8de]{
 -- Page 235, Offset 43750.
 -- ちなみに千年城が消滅したのは
-Incidentally, the Millennium Castle dissipated
+Incidentally, the reason her Millennium Castle dissipated%{n}
 }
 [sha:7d550cf29b4de2b01fa65f4a7e2ae319fe83f570]{
 -- Page 235, Offset 43751.
 -- 彼女の空想具現化のランクが下がったためです。
-because her Marble Phantasm rank was reduced as a result.
+is because her Marble Phantasm rank dropped a notch.
 }
 [sha:096385fb702b5dd2b80fa7da0d394cd02b8e96f6]{
 -- Page 236, Offset 43752.
 -- 『光体』状態の真祖では空想具現化は
-The True Ancestor in the 'Luminary' state can
+A True Ancestor in a 'Luminary' state can%{n}
+only
 }
 [sha:93def1e370938010fbc01d0dbd86534acf180a53]{
 -- Page 236, Offset 43753.
 -- 大雑把なものしかできません。
-only produce a rough version of a Marble Phantasm.
+ produce a rough sketch of a Marble%{n}
+Phantasm.
 }
 [sha:0633c3074bea18267440c83dfffd931ff313152b]{
 -- Page 237, Offset 43754.
 -- 出力が大きくなった分、きめ細やかな空想を
-The larger the output, the harder it is to
+The larger the output,
 }
 [sha:53afbfbb25032d1bf309f243800fe59c8209dd8b]{
 -- Page 237, Offset 43755.
 -- 構築できなくなったのです。
-meticulously construct a phantasm accurately.
+ the harder it becomes to create%{n}
+finer details on the phantasms.
 }
 [sha:6c466f2664c7c1a8e412544fcf41d60d642d18dc]{
 -- Page 238, Offset 43756.
 -- 『光体』状態の彼女にできる事は、
-The things Arcueid can do in her 'Luminary' state
+The things Arcueid is capable of in her%{n}
+'Luminary' state
 }
 [sha:63b1f1fc140afd7c4ca8f3820a2b8eb5108c4453]{
 -- Page 238, Offset 43757.
 -- 超広範囲の魔眼と、
-include the use of her Ultra-wide Mystic Eyes,
+ include the use of her%{n}
+Ultra-wide Mystic Eyes―――
 }
 [sha:35c1cd278b8c1d98e24d409b2205d7379a983355]{
 -- Page 239, Offset 43758.
 -- 自身に向けられる攻撃をキャンセルする空想無量化と、
-canceling attacks directed at her with Imaginary Nullification,
+Canceling attacks directed at her using Phantasmal%{n}
+Nullification,
 }
 [sha:eda8f1c830acb99d5334fdc67df9d0e4598f7232]{
 -- Page 239, Offset 43759.
 -- 地熱操作、大気操作、といったものになります。
-manipulate geothermal heat, the atmosphere, and so on.
+ the manipulation of geothermal heat,
+ the%{n}manipulation of the atmosphere,
+ and so on.
```

Comment:
Bunch of stuff from the mats (which I TL'd)
some of it just isn't quite right either.

>The abilities Arcueid possess in her Luminary state are her ultrawide-range Mystic Eyes,
>her Fantasy Dequantification, which allows her to cancel incoming attacks,
>and geothermal manipulation and atmospheric manipulation.